self-hosting/.env.example

```
@@ -0,0 +1,41 @@
+
+# Database. DATABASE_URL is inferred from these values inside docker-compose.
+POSTGRES_USER=instant
+POSTGRES_PASSWORD=pleasegenerateapasswordhere
+POSTGRES_DB=instant
+
+# Object storage. Server AWS credentials are inferred from these MinIO values.
+# If using AWS S3, set MINIO_ROOT_USER to your AWS Access Key ID
+# and MINIO_ROOT_PASSWORD to your AWS Secret Access Key, and
+# leave S3_ENDPOINT and S3_PUBLIC_ENDPOINT blank.
+MINIO_ROOT_USER=minioadmin
+MINIO_ROOT_PASSWORD=minioadmin
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MINIO_ROOT_PASSWORD key should go before the MINIO_ROOT_USER key

(UnorderedKey)